### DISCUSSION TITLE
Welcome to Coordinate-System Discussions!

### DISCUSSION THREAD

**Discussion by panguojun** [panguojun/Coordinate-System#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Powe12** [comment]
Nice

**Reply by panguojun** [reply]
Thank you for your participation. Do you want to have a try?
Python version:
Python 3.11,
pip install coordinate_system

**Comment by Powe12** [comment]
Nice dear welcome to you dear

**Reply by panguojun** [reply]
Thank you, I'm glad you participated. I think you can try the Python version and the visualization tools.

**Comment by Powe12** [comment]
I want to work with you and your team please

**Reply by panguojun** [reply]
for Python311  pip install :
pip install coordinate_system
pip install phg_vis

test:
from coordinate_system import vec3, quat, coord3
from phg import vis

# sample
a = coord3(1,0,0)
b = coord3(0,1,0)
print((b / a).to_string())

# vis sample
phg_string = "coord(1,0,0);"
vis(phg_string);
